src/ui/src/app/dashboard/dashboard-body/side-nav/side-nav.component.ts

```diff
@@ -13,5 +13,8 @@ import { EmployeeService } from '../../../service/employee.service';
 export class SideNavComponent {
   userService=inject(WolfDenService)
   employeeService=inject(EmployeeService)
-
+  
+ngOnInit(){
+  console.log('hi',this.userService.role)
```

Comment:
?

Reply:
There was an issue in getting userService.role. now it is removed. it was used for checking.